ext/openssl/ossl_ssl.c

```diff
@@ -1557,7 +1577,11 @@ ossl_start_ssl(VALUE self, int (*func)(), const char *funcname, VALUE opts)
                 continue;
 #endif
 	    if (errno) rb_sys_fail(funcname);
-	    ossl_raise(eSSLError, "%s SYSCALL returned=%d errno=%d state=%s", funcname, ret2, errno, SSL_state_string_long(ssl));
+        peeraddr_info = peeraddr_ip_str(self);
+        peeraddr = RTEST(peeraddr_info) ? StringValuePtr(peeraddr_info) : "(null)";
+	    ossl_raise(eSSLError, "%s SYSCALL returned=%d errno=%d peeraddr=%s state=%s",
+                funcname, ret2, errno, peeraddr, SSL_state_string_long(ssl));
```

Comment:
Indentation is broken in this `switch`. Please set hard tab width to 8 in your editor and adjust so that it looks aligned. (New code uses spaces, but old code still has tabs.)

Reply:
Should I just fix this entire method to use the new spaces indentation? I tried using tab with width 8 only on the switch statement, but honestly it doesn't look a lot better.

Reply:
The code now looks consistent and good for me on vim with `set tabstop=8` (the default for C in most text editors, I think).

Existing code with mixed tabs can be left as-is for now, and we will gradually update to spaces only as we modify actual code. I tend to avoid making a commit changing styles only since doing that would mess up `git blame`.